nbconvert/nbconvertapp.py

```diff
@@ -138,6 +138,11 @@ def _classes_default(self):
             the exporter, and is usually '.nbconvert'."""
     ).tag(config=True)
 
+    output_files_dir = Unicode('%s_files',
```

Comment:
Can you use format-strings, so this would be `'{notebook_name}_files'`, rather than `%s`? Below would then become `output_files_dir.format(notebook_name=notebook_name)`